timesketch/api/v1/resources/archive.py

```diff
@@ -462,58 +482,142 @@ def _unarchive_sketch(self, sketch: Sketch):
         return HTTP_STATUS_CODE_OK
 
     def _archive_sketch(self, sketch: Sketch):
```

Comment:
Maybe we can mention that this is a revertable/non-destructive action in the doc string?